### DISCUSSION TITLE
Snacks Input config doesnt change vim.ui.input

### DISCUSSION THREAD

**Discussion by drewfowler** [folke/snacks.nvim#380] [Q&A] [answered]
I can't get snacks input to change the vim.ui.input. My other snack configurations work just fine. 
I assume this is due to ignorance on my part, but would love some help.

The current input config is just to see if anything changed. This is what it currently looks like. 
![image](https://github.com/user-attachments/assets/bd48fd24-7e87-4e7e-ae1f-11c8dce471c5)

I also checked snacks health and everything is ok. 

Here is my snacks.lua file. 
`return {
  {
    "folke/snacks.nvim",
    priority = 1000,
    lazy = false,
    ---@type snacks.Config
    opts = {
      bigfile = { enabled = true },
      notifier = { enabled = true },
      quickfile = { enabled = true },
      statuscolumn = { enabled = true },
      words = { enabled = true },
      dashboard = {
        enabled = true,
        sections = {
          { section = "header" },
          { icon = " ", title = "Keymaps", section = "keys", indent = 2, padding = 1 },
          { icon = " ", title = "Recent Files", section = "recent_files", indent = 2, padding = 1 },
          { icon = " ", title = "Projects", section = "projects", indent = 2, padding = 1 },
          { section = "startup" },
        },
      },
      terminal = {
        enabled = true,
        win = { position = "right", width = 0.25 },
      },
      input = {
        enabled = true,
        prompt_pos = "right",
        icon_pos = "right",
      },
    },
  },
}`

**Comment by dpetka2001** [comment, marked as answer]
https://github.com/folke/snacks.nvim/issues/239#issuecomment-2557396734
